examples/with-couchbase/README.md

```diff
@@ -1,95 +0,0 @@
-## Example app using Couchbase
```

Comment:
Was this meant to be fully deleted? Seems there's a couple others as well like `with-mocha`, `with-sentry`, and `with-zones`

Reply:
Great catch, thank you! This was not intentional, fixing now.